sync-pipelines-rust/src/settings.rs

```diff
@@ -0,0 +1,69 @@
+use config::{Config, ConfigError, File};
+use serde::Deserialize;
+use std::path::PathBuf;
+
+#[derive(Deserialize)]
+pub struct LastSyncTime {
+    start_time: StartTime,
+    files: SyncFiles,
+}
+
+#[derive(Deserialize)]
+pub struct SyncFiles {
+    token_transfers: PathBuf,
+    transactions: PathBuf,
+}
+
+#[derive(Deserialize)]
+pub struct StartTime {
+    token_transfers: String,
+    transactions: String,
+}
+
+#[derive(Deserialize)]
+pub struct Settings {
+    chain: String,
+    last_sync_time: LastSyncTime,
+    native_currency: String,
+    transaction_id_prefix: String,
+}
+
+const CONFIG_FILE_PATH: &str = "./Config.toml";
+
+impl Settings {
+    pub fn new() -> Result<Self, ConfigError> {
+        let s = Config::builder()
+            .add_source(File::with_name(CONFIG_FILE_PATH).required(false))
```

Comment:
```suggestion
            .add_source(File::with_name(CONFIG_FILE_PATH).required(true))
```
we need a file for config